### PR TITLE
Shell scripts

### PR DESCRIPTION
**Pull request by airbagy**

### CODE REVIEW

File: scripts/install_fonts.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
```
Comment (FrankYFTang) [inline scripts/install_fonts.sh:3]: Add
```
# Copyright 2020 Google LLC
```

File: scripts/setup.sh [hunk 2]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
```
Comment (FrankYFTang) [inline scripts/setup.sh:3]: Add
```
# Copyright 2020 Google LLC
```

File: scripts/setup.sh [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+# Set up script for dependencies and required libraries
+# 1. Install libfreetype6 libcairo2
+# 2. Install qahirah (cairo) wraper and dependencies
+# 3. Install other required python libraries
+
+apt-get -y update
+apt-get -y install libfreetype6 libcairo2 libsm6 libxext6 libfontconfig1 libxrender1 fontconfig
+
```
Comment (FrankYFTang) [inline scripts/setup.sh:11]: add
mkdir -p qahirah
mkdir -p pybidi
mkdir -p python_freetype

File: scripts/start.sh [hunk 4]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
```
Comment (FrankYFTang) [inline scripts/start.sh:3]: Add
```
# Copyright 2020 Google LLC
```

**Review by FrankYFTang — APPROVED** [review]
Make those changes and merge after.

File: scripts/setup.sh (hunk 3 above)
Comment (airbagy) [reply scripts/setup.sh:11]: Or maybe it is better to add `git submodule update --init --recursive`

**Pull request merged**